src/System.Reflection.Emit.Lightweight/ref/System.Reflection.Emit.Lightweight.cs

```diff
@@ -8,6 +8,29 @@
 
 namespace System.Reflection.Emit
 {
+    public sealed partial class DynamicILInfo
+    {
+        internal DynamicILInfo(object scope, DynamicMethod method, byte[] methodSignature) { }
+        public System.Reflection.Emit.DynamicMethod DynamicMethod { get { throw null; } }
+        public int GetTokenFor(byte[] signature) { throw null; }
+        public int GetTokenFor(DynamicMethod method) { throw null; }
+        public int GetTokenFor(RuntimeFieldHandle field) { throw null; }
+        public int GetTokenFor(RuntimeFieldHandle field, RuntimeTypeHandle contextType) { throw null; }
+        public int GetTokenFor(RuntimeMethodHandle method) { throw null; }
+        public int GetTokenFor(RuntimeMethodHandle method, RuntimeTypeHandle contextType) { throw null; }
+        public int GetTokenFor(RuntimeTypeHandle type) { throw null; }
+        public int GetTokenFor(string literal) { throw null; }
```

Comment:
Tests for all these `GetTokenFor` methods?

Reply:
in my TODO list.

Reply:
Added test for `GetTokenFor` methods took from full framework tests.